cmake/multi_image.cmake

```diff
@@ -96,8 +95,15 @@ function(add_child_image_from_source)
     get_board_without_ns_suffix(${BOARD} ACI_BOARD)
   endif()
 
-  set(${ACI_NAME}_BOARD ${ACI_BOARD})
+  if (NOT ACI_DOMAIN AND DOMAIN)
+    # If no domain is specified, a child image will inherit the domain of
+    # its parent.
+    message("SETTING DOMAIN")
```

Comment:
Seeing the message `SETTING DOMAIN` gives no useful information to the end-user.

Setting it to what ?
and precisely why is this important for the end-user to know, compared to a lot of other nice to have info. 
The `ACI_DOMAIN` is already printed below, so consider adding `(inherited)` there instead.
```
if(...)
  set(ACI_DOMAIN_INHERITED "(inherited)")
endif()
...
message("\n=== child image ${ACI_NAME} - ${ACI_DOMAIN} ${ACI_DOMAIN_INHERITED} begin ===")
```

Reply:
Fixed, but i used `set(inherited ...)` instead, since ACI_ is the prefix for the arguments.